cli/src/app.rs

```diff
@@ -312,11 +310,8 @@ pub fn app<'a>() -> clap::Command<'a> {
                 ),
         );
 
-    #[cfg(feature = "extensions")]
-    {
-        app = app.subcommand(extensions::command());
-        app = extensions::add_extensions_subcommands(app);
-    }
+    app = app.subcommand(extensions::command());
```

Comment:
We can move this directly into the builder, right?

Reply:
Yes -- though I'd leave the line after that as-is, outside of the builder.